ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -87,36 +89,83 @@ class DateTimePickerDemoController: DemoController {
         dateTimePicker.present(from: self, with: .dateTime, startDate: startDate ?? Date(), datePickerType: datePickerType)
     }
 
+    func calcDatePickerParams() -> (startDate: Date, endDate: Date, titles: DateTimePicker.Titles) {
+        let startDate = startDate ?? Date()
+        let endDate = endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
```

Comment:
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift:93:25: error: variable used within its own initial value
        let startDate = startDate ?? Date()
                        ^
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift:94:23: error: variable used within its own initial value
        let endDate = endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
                      ^

Touch /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/FluentUI.framework (in target 'FluentUI' from project 'FluentUI')
    cd /Users/runner/work/fluentui-apple/fluentui-apple/ios
    /usr/bin/touch -c /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/FluentUI.framework

WriteAuxiliaryFile /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Intermediates.noindex/FluentUI.Demo.build/Release-iphonesimulator/FluentUI.Demo.build/Objects-normal/x86_64/FluentUI.Demo.LinkFileList (in target 'FluentUI.Demo' from project 'FluentUI.Demo')
    cd /Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo
    write-file /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Intermediates.noindex/FluentUI.Demo.build/Release-iphonesimulator/FluentUI.Demo.build/Objects-normal/x86_64/FluentUI.Demo.LinkFileList

WriteAuxiliaryFile /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Intermediates.noindex/FluentUI.Demo.build/Release-iphonesimulator/FluentUI.Demo.build/Objects-normal/arm64/FluentUI.Demo.LinkFileList (in target 'FluentUI.Demo' from project 'FluentUI.Demo')
    cd /Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo
    write-file /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Intermediates.noindex/FluentUI.Demo.build/Release-iphonesimulator/FluentUI.Demo.build/Objects-normal/arm64/FluentUI.Demo.LinkFileList

PBXCp /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/AppCenter_AppCenterDistribute.bundle /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/FluentUI.Demo.app/AppCenter_AppCenterDistribute.bundle (in target 'FluentUI.Demo' from project 'FluentUI.Demo')
    cd /Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/AppCenter_AppCenterDistribute.bundle /Users/runner/Library/Developer/Xcode/DerivedData/FluentUI-cunfevvrcewzjwgrlndrwmvshksg/Build/Products/Release-iphonesimulator/FluentUI.Demo.app

** BUILD FAILED **

Reply:
Thank you for your help.
Three points here:
1) I have already fixed the code; the Push is not working now, but I hope that by the time you read this, it will be ok.
2) I'm not a Swift expert, but the above lines seem to be correct to me and in fact they compile in Xcode 12.5 on my machine. For example in the first line I declare a new variable startDate and I use the startDate parameter to initialize it.
3) So there might be some difference with my local dev env. I have installed the linter and also run 'pod lib lint' locally. Do I miss anything?